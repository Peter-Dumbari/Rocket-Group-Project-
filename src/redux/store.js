import { configureStore } from '@reduxjs/toolkit';
import rocketSlice from './Rocket/rocketSlice';
import missionReducer from './Mission/missionSlice';

const store = configureStore({
  reducer: {
    rockets: rocketSlice,
    mission: missionReducer,
  },
});

export default store;
