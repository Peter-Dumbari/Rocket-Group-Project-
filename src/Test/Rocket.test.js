import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Rocket from '../Components/Rockect';

describe('Component test snapshot', () => {
  it('Rockets renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Rocket />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
