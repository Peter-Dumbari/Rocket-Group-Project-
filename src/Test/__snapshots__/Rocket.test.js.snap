// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component test snapshot Rockets renders correctly 1`] = `"no rockets"`;
